### PR TITLE
Add parameter description in 2012 version

### PR DESCRIPTION
**Pull request by velkovb**
Added parameter description in 2012 version and adjusted formatting in all versions so it is uniform.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
Looks good.

**Comment by tseward** [comment]
Looks good.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 1]
```diff
@@ -29,33 +29,29 @@ Get-ADDomainControllerPasswordReplicationPolicy [-AuthType <ADAuthType>] [-Crede
 ```
 
 ## DESCRIPTION
-The Get-ADDomainControllerPasswordReplicationPolicy gets the users, computers, service accounts and groups that are members of the applied list or denied list for a read-only domain controller's (RODC) password replication policy.
-To get the members of the applied list, specify the AppliedList parameter.
-To get the members of the denied list, specify the DeniedList parameter.
+The **Get-ADDomainControllerPasswordReplicationPolicy** cmdlet gets the users, computers, service accounts and groups that are members of the applied list or denied list for a read-only domain controller's (RODC) password replication policy.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
The **Get-ADDomainControllerPasswordReplicationPolicy** cmdlet gets the users, computers, service accounts, and groups that are members of the applied list or denied list for a read-only domain controller's (RODC) password replication policy.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 2]
```diff
@@ -29,33 +29,29 @@ Get-ADDomainControllerPasswordReplicationPolicy [-AuthType <ADAuthType>] [-Crede
...
-For example, you can use the Get-ADDomainController cmdlet to retrieve a domain controller object and then pass the object through the pipeline to the Get-ADDomainControllerPasswordReplicationPolicy cmdlet.
+You can also set the *Identity* parameter to a domain controller object variable, such as `$<localDomainControllerObject>`, or pass a domain controller object through the pipeline operator to the *Identity* parameter.
+For example, you can use the **Get-ADDomainController** cmdlet to retrieve a domain controller object and then pass the object through the pipeline operator to the **Get-ADDomainControllerPasswordReplicationPolicy** cmdlet.
 
 If you specify a writeable domain controller for this cmdlet, the cmdlet returns a non-terminating error.
 
 ## EXAMPLES
 
-### -------------------------- EXAMPLE 1 --------------------------
+### Example 1: gets from an RODC domain controller password replication policy the allowed accounts showing the name and object class of each
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
### Example 1: Gets the allowed accounts from an RODC domain controller password replication policy, showing the name and object class of each.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 3]
```diff
@@ -29,33 +29,29 @@ Get-ADDomainControllerPasswordReplicationPolicy [-AuthType <ADAuthType>] [-Crede
...
 ```
 C:\PS>Get-ADDomainControllerPasswordReplicationPolicy -Identity "FABRIKAM-RODC1" -Allowed | ft Name,ObjectClass
 ```
 
-Description
+This command gets from an RODC domain controller password replication policy the allowed accounts showing the name and object class of each.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This command gets the allowed accounts from an RODC domain controller password replication policy, showing the name and object class of each.
```
Suggestion only.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 4]
```diff
@@ -122,7 +136,14 @@ Accept wildcard characters: False
 ```
 
 ### -Denied
-{{Fill Denied Description}}
+Specifies the users, computers, groups or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this read-only domain controller (RODC).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
Specifies the users, computers, groups, or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this read-only domain controller (RODC).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 5]
```diff
@@ -122,7 +136,14 @@ Accept wildcard characters: False
...
+You can specify more than one value by using a comma-separated list.
+The acceptable values for this parameter are:
+
+- A distinguished name
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A distinguished name.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 6]
```diff
@@ -122,7 +136,14 @@ Accept wildcard characters: False
...
+- A GUID (objectGUID)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A GUID (objectGUID). 
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 7]
```diff
@@ -122,7 +136,14 @@ Accept wildcard characters: False
...
 
 ### -Denied
-{{Fill Denied Description}}
+Specifies the users, computers, groups or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this read-only domain controller (RODC).
+You can specify more than one value by using a comma-separated list.
+The acceptable values for this parameter are:
+
+- A distinguished name
+- A GUID (objectGUID) 
+- A security identifier (objectSid)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A security identifier (objectSid).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 8]
```diff
@@ -122,7 +136,14 @@ Accept wildcard characters: False
...
 ### -Denied
-{{Fill Denied Description}}
+Specifies the users, computers, groups or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this read-only domain controller (RODC).
+You can specify more than one value by using a comma-separated list.
+The acceptable values for this parameter are:
+
+- A distinguished name
+- A GUID (objectGUID) 
+- A security identifier (objectSid) 
+- A SAM account name (sAMAccountName)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A SAM account name (sAMAccountName).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 9]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-{{Fill Identity Description}}
+Specifies an Active Directory domain controller object by providing one of the following values.
+The identifier in parentheses is the Lightweight Directory Access Protocol (LDAP) display name for the attribute. 
+The acceptable values for this parameter are:
+
+- A GUID (objectGUID)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A GUID (objectGUID).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 10]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-{{Fill Identity Description}}
+Specifies an Active Directory domain controller object by providing one of the following values.
+The identifier in parentheses is the Lightweight Directory Access Protocol (LDAP) display name for the attribute. 
+The acceptable values for this parameter are:
+
+- A GUID (objectGUID) 
+- An IPV4Address
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- An IPV4Address.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 11]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
+- A Global IPV6Address
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A Global IPV6Address.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 12]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
 ### -Identity
-{{Fill Identity Description}}
+Specifies an Active Directory domain controller object by providing one of the following values.
+The identifier in parentheses is the Lightweight Directory Access Protocol (LDAP) display name for the attribute. 
+The acceptable values for this parameter are:
+
+- A GUID (objectGUID) 
+- An IPV4Address
+- A Global IPV6Address 
+- A DNS Host Name (dNSHostName)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A DNS Host Name (dNSHostName).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 13]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
-{{Fill Identity Description}}
+Specifies an Active Directory domain controller object by providing one of the following values.
+The identifier in parentheses is the Lightweight Directory Access Protocol (LDAP) display name for the attribute. 
+The acceptable values for this parameter are:
+
+- A GUID (objectGUID) 
+- An IPV4Address
+- A Global IPV6Address 
+- A DNS Host Name (dNSHostName) 
+- A name of the server object
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A name of the server object.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 14]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
+- A Distinguished Name of the NTDS Settings object
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A Distinguished Name of the NTDS Settings object.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 15]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
+- A distinguished name of the server object that represents the domain controller
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A distinguished name of the server object that represents the domain controller.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 16]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
+- A GUID of NTDS settings object under the configuration partition
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A GUID of NTDS settings object under the configuration partition.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 17]
```diff
@@ -137,7 +158,25 @@ Accept wildcard characters: False
...
+- A GUID of server object under the configuration partition
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- A GUID of server object under the configuration partition.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 18]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
 ```
 
 ### -Server
-{{Fill Server Description}}
+Specifies the Active Directory Domain Services instance to connect to, by providing one of the following values for a corresponding domain name or directory server.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
Specifies the Active Directory Domain Services instance to connect to by providing one of the following values for a corresponding domain name or directory server.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 19]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+The service may be any of the following: Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory snapshot instance.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
The service may be any of the following: Active Directory Lightweight Domain Services, Active Directory Domain Services, or Active Directory snapshot instance.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 20]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+
+Specify the Active Directory Domain Services instance in one of the following ways: 
+
+Domain name values:
+
+- Fully qualified domain name
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- Fully qualified domain name.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 21]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+- NetBIOS name
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- NetBIOS name.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 22]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+
+Directory server values: 
+
+- Fully qualified directory server name
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- Fully qualified directory server name.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 23]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+- NetBIOS name
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- NetBIOS name.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 24]
```diff
@@ -152,7 +191,27 @@ Accept wildcard characters: False
...
+- Fully qualified directory server name and port
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
- Fully qualified directory server name and port.
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 25]
```diff
@@ -167,12 +226,12 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 26]
```diff
@@ -181,13 +240,13 @@ Returns one or more objects that represent the users, computers, service account
 The list returned depends on the parameters specified.
 
 ## NOTES
-* This cmdlet does not work with AD LDS.
-
-  This cmdlet does not work when targeting a snapshot using the Server parameter.
+* This cmdlet does not work with Active Directory Lightweight Directory Services.
+* This cmdlet does not work when targeting a snapshot using the Server parameter..
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
* This cmdlet does not work when targeting a snapshot using the Server parameter.
```

File: docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 27]
```diff
@@ -238,12 +234,12 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 28]
```diff
@@ -126,8 +122,8 @@ If the cmdlet is run from such a provider drive, the account associated with the
 To specify this parameter, you can type a user name, such as User1 or Domain01\User01 or you can specify a **PSCredential** object.
 If you specify a user name for this parameter, the cmdlet prompts for a password.
 
-You can also create a **PSCredential** object by using a script or by using the Get-Credentialhttp://go.microsoft.com/fwlink/?LinkID=293936 cmdlet.
-You can then set the **Credential** parameter to the **PSCredential** object.
+You can also create a **PSCredential** object by using a script or by using the **Get-Credential** cmdlet.
```
Comment (e0i) [inline docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md]: The link is valid. Please turn it into a proper Markdown link. Thanks.

**Comment by rjagiewich** [comment]
Closing and reopening to try to force the build

**Pull request merged**